pkg/crc/machine/bundle/copier.go

```diff
@@ -61,41 +61,26 @@ func (copier *Copier) CachedPath() string {
 func (copier *Copier) CopyPrivateSSHKey(srcPath string) error {
 	sshKeyFileName := filepath.Base(copier.srcBundle.GetSSHKeyPath())
 	destPath := copier.resolvePath(sshKeyFileName)
-	if err := crcos.CopyFileContents(srcPath, destPath, 0400); err != nil {
-		return err
-	}
-	// copier.srcBundle.ClusterInfo.SSHPrivateKeyFile = sshKeyFileName
-	return nil
+	return crcos.CopyFileContents(srcPath, destPath, 0400)
 }
 
 func (copier *Copier) SetKubeAdminPassword(kubePassword string) error {
 	kubeAdminPasswordFileName := copier.srcBundle.ClusterInfo.KubeadminPasswordFile
 	path := copier.resolvePath(kubeAdminPasswordFileName)
-	if err := ioutil.WriteFile(path, []byte(kubePassword), 0600); err != nil {
-		return err
-	}
-	// copier.copiedBundle.ClusterInfo.KubeadminPasswordFile = kubeAdminPasswordFileName
-	return nil
+	return ioutil.WriteFile(path, []byte(kubePassword), 0600)
 }
 
 func (copier *Copier) CopyKubeConfig() error {
 	kubeConfigFileName := filepath.Base(copier.srcBundle.GetKubeConfigPath())
 	srcPath := copier.srcBundle.GetKubeConfigPath()
 	destPath := copier.resolvePath(kubeConfigFileName)
-	if err := crcos.CopyFileContents(srcPath, destPath, 0640); err != nil {
-		return err
-	}
-	// copier.copiedBundle.ClusterInfo.KubeConfig = kubeConfigFileName
-	return nil
+	return crcos.CopyFileContents(srcPath, destPath, 0640)
 }
 
 func (copier *Copier) CopyOcExecutable() error {
 	srcPath := copier.srcBundle.GetOcPath()
 	destPath := copier.resolvePath(constants.OcExecutableName)
```

Comment:
Can you use use `filepath.Base(srcPath)` instead of `constants.OcExecutableName`?